readme.md

```diff
@@ -2,6 +2,13 @@
 
 Copyright (C) 2011-2012 Steve Donovan.
 
+## Fork of Steve Donovan's LDoc that fixes issue 301 and adds the following tags:
```

Comment:
Before we consider merging this, you should remove anything that suggests this is a fork. We're merging to the bonefied original here ;-)